data/dfndata/npc/undead.dfn

```diff
@@ -479,3 +492,31 @@ NAME=#//a ghoul
 ID=0x0099
 SKIN=0x0
 }
+
+//lady of the snow - Base NPC with shared DFN tags
+[base_lady_of_the_snow]
+{
+GET=base_undead
+NAME=#//lady of the snow
+ID=0x0fc
+ORIGIN=se
+}
+
+[ladyofthesnow]
+{
+GETSE=ladyofthesnow_se
```

Comment:
Add one of these for each expansion, so adding NPC with wrong expansion selected doesn't break the server. We can update where these point at when we have definitions for different eras:

GETUO=ladyofthesnow_se
GETT2A=ladyofthesnow_se
GETUOR=ladyofthesnow_se
GETTD=ladyofthesnow_se
GETLBR=ladyofthesnow_se
GETAOS=ladyofthesnow_se
GETSE=ladyofthesnow_se
GETML=ladyofthesnow_se
GETSA=ladyofthesnow_se
GETHS=ladyofthesnow_se
GETTOL=ladyofthesnow_se